### PR TITLE
Create security-authorization-of-web-endpoints-reference.md

### PR DESCRIPTION
**Pull request by MichalMaler**
Renaming file to match the documentation changes
https://github.com/quarkusio/quarkus/pull/30549

** If you are updating a guide, please submit your pull request to the main repository: https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc **

### CODE REVIEW

File: _redirects/guides/security-authorization-of-web-endpoints-reference.adoc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[source,markdown]
+---
+permalink: /guides/security-authorization-of-web-endpoints-reference.adoc/index.html
+newUrl: /guides/security-authorize-web-endpoints-reference.adoc
```
Comment (michelle-purcell) [inline _redirects/guides/security-authorization-of-web-endpoints-reference.adoc:4]: The `.adoc' file extension bit is not needed. 

```suggestion
newUrl: /guides/security-authorize-web-endpoints-reference
```

[Seee demo and slides]

File: _redirects/guides/security-authorization-of-web-endpoints-reference.adoc [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[source,markdown]
+---
+permalink: /guides/security-authorization-of-web-endpoints-reference.adoc/index.html
```
Comment (michelle-purcell) [inline _redirects/guides/security-authorization-of-web-endpoints-reference.adoc:3]: The `.adoc' file extension bit is not needed.
```suggestion
permalink: /guides/security-authorization-of-web-endpoints-reference/index.html
```
[See demo  slides & example PR]

**Comment by michelle-purcell** [comment]
@MichalMaler -  The original pre-diataxis published version topic was called `security-authorization.adoc'.

A redirect file named `security-authorization.md` will also be needed for the original source page URL. 
Thanks

**Comment by maxandersen** [comment]
is this still relevant?

**Comment by MichalMaler** [comment]
> is this still relevant?

Hello, Max,
yes, it is. Let me apply these coments.

**Comment by rolfedh** [comment]
Hi @MichalMaler, I noticed the file uses an `.adoc` file extension, while the other most recent updated files use `.md`, so I recommend updating it to use `.md` instead.
The contents of the file follow the same structure as the other files in the `_redirects/guides` directory. So the redirect should function correctly once it's deployed to the community website.

**Review by rolfedh — APPROVED** [review]

**Comment by rolfedh** [comment]
LGTM

**Comment by MichalMaler** [comment]
@maxandersen @jmartisk Hello! Can one of you merge this? :)

**Review by jmartisk — APPROVED** [review]

**Pull request merged**